lib/Zonemaster/CLI.pm

```diff
@@ -597,10 +603,17 @@ sub add_fake_delegation {
         my ( $name, $ip ) = split( '/', $pair, 2 );
 
         if ( not $name ) {
-            say STDERR "--ns must have be a name or a name/ip pair.";
+            say STDERR __( "--ns must be a name or a name/ip pair." );
             exit( 1 );
         }
 
+        if ( $name =~ m/\.\./i ) {
+            say STDERR __x( "The name of the nameserver '{ns}' contains consecutive dots.", ns => $name );
```

Comment:
In a msgid in Zonemaster-Engine, an argument for this type must be "nsname". It would be good to use the same definitions here because it makes it easier for the translator. See https://github.com/zonemaster/zonemaster-engine/blob/develop/docs/logentry_args.md